src/index.html

```diff
@@ -13,18 +13,18 @@ <h1>Leaderboard</h1>
         <div class="score">
             <div class="scoreHeader">
                 <h2>Recent Scores</h2>
-                <button>Refresh</button>
+                <button type="submit" id="refresh">Refresh</button>
             </div>
             <div class="recentScore" id="recentScores"> 
             <ul class="scoreList" id="scoreList"> </ul> 
             </div>       
         </div>
         <div class="userInput">
             <h2>Add your scrore</h2>
-            <form action="#" class="form">
+            <form action="#" class="form" id="form">
                 <input class="formItem" type="text" name="yourName" id="name">
                 <input class="formItem" type="text" name="yourScore" id="score">
```

Comment:
[OPTIONAL] I think you could validate inputs to avoid users posting impertinent data (By posting strings or empty values to number fields). This might help avoid bugs in the future and will also bring a better UX.

Another good idea could be thinking about accessibility. Keeping it accessible by adding `label` tags, etc. could be really pertinent.

Reply:
Thanks for the good suggestion. These will be implemented in the new milestone.